### ISSUE TITLE
GUI client works, CLI client does not

### ISSUE THREAD

**Issue by geekraver** [synergy/synergy#5091]
I am running the server on a Mac and client on Ubuntu. I have no issue if I run the GUI client on Ubuntu;
however I want to run the CLI client as part of my login script. When I use that I just get repeated connect/disconnect cycling. Client logs:

[2015-11-07T11:39:42] NOTE: connecting to '192.168.2.2': 192.168.2.2:24800
[2015-11-07T11:39:42] DEBUG1: connecting to server
[2015-11-07T11:39:42] DEBUG1: registered event type connected as 10
[2015-11-07T11:39:42] DEBUG1: registered event type connectionFailed as 11
[2015-11-07T11:39:42] DEBUG: event queue is ready
[2015-11-07T11:39:42] DEBUG1: registered event type inputReady as 12
[2015-11-07T11:39:42] DEBUG1: registered event type inputShutdown as 13
[2015-11-07T11:39:42] DEBUG1: connected;  wait for hello
[2015-11-07T11:39:42] DEBUG1: registered event type disconnected as 14
[2015-11-07T11:39:42] DEBUG1: registered event type outputError as 15
[2015-11-07T11:39:42] DEBUG1: registered event type outputShutdown as 16
[2015-11-07T11:39:42] DEBUG1: registered event type stopRetry as 17
[2015-11-07T11:39:56] DEBUG1: disconnected
[2015-11-07T11:39:56] NOTE: disconnected from server
[2015-11-07T11:39:56] DEBUG: retry in 1 seconds

On the Mac I see I just see this over and over:

[2015-11-07T11:44:33] NOTE: accepted client connection
[2015-11-07T11:44:45] WARNING: failed to accept secure socket

**Comment by XinyuHou** [comment]
What command do you use to start Synergy client?

**Comment by geekraver** [comment]
synergyc -f -d DEBUG1 192.168.2.2

**Comment by XinyuHou** [comment]
This looks like a support ticket. Please contact our help desk or forum.
http://synergy-project.org/forum/
https://synergy-project.org/contact/

**Comment by geekraver** [comment]
I did that. That seems like a black hole as I never got any response.

**Comment by jesseconnell** [comment]
Is this still a black hole?  I have a similar issue.

**Comment by jesseconnell** [comment]
I got mine working.  I first made sure client/server names were all same case (lower).  Highly doubt that worked, but I did that, so I'm saying it.  Then I saw in server configuration I could uncheck SSL.  (I don't need SSL, so that was acceptable).  I bounced everything and it came up working.

I saw nothing about SSL in the client, but then my client is version 1.3.6.  Is SSL support a new thing, turned on by default?

**Comment by geekraver** [comment]
Yes, it works with SSL disabled, but that isn't very satisfactory. I don't see any client-side option to enable SSL; presumably the server is expecting it and the client is not doing it.

**Comment by dinomite** [comment]
I get this on Mac.  How do you disable SSL?

```
[2015-11-23T10:03:46] INFO: starting server
[2015-11-23T10:03:46] INFO: config file: /var/folders/77/xjt9xv456nzfx5y1vwsq80480000gn/T/Synergy.ux9644
[2015-11-23T10:03:46] INFO: log level: INFO
[2015-11-23T10:03:46] INFO: drag and drop enabled
[2015-11-23T10:03:46] NOTE: started server, waiting for clients
2015-11-23 10:03:46.306 synergys[9650:761418] starting cocoa loop
[2015-11-23T10:06:38] NOTE: accepted client connection
[2015-11-23T10:06:38] ERROR: ssl error occurred (system call failure)
[2015-11-23T10:06:38] WARNING: failed to accept secure socket
```

**Comment by saxtell-cb** [comment]
I had this same issue after upgrading to 1.7.5. When I unchecked the SSL option on the client, everything worked. Then I tried checking SSL on both client (Edit -> Settings) and server (Edit -> Settings) and everything worked as well. Seems that my client didn't have SSL selected by default and my server did. You just have to match them up.

**Comment by ivany4** [comment]
Is this gonna be fixed anytime soon? I have exactly the same setup and exactly the same issues.

**Comment by the-wes** [comment]
Has anyone tried running synergyc with --enable-crypto and --profile-dir set according to https://github.com/symless/synergy/wiki/Plugins ?
